### PR TITLE
32 use the updated refarch prettier config

### PR DESCRIPTION
**Pull request by langehm**
Issues #32

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
-import MucButton from './Button/MucButton.vue'
-import MucBanner from './Banner/MucBanner.vue'
-import MucIntro from './Intro/MucIntro.vue'
-import { MucCallout } from './Callout'
```
Comment (FabianWilms) [inline src/components/index.ts:4]: You deleted your own new component here ;)

File: docs/components/muc-banner.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <script setup>
-import Basic from './demo/MucBanner/Basic.vue'
+import Basic from './demo/MucBanner/BasicExample.vue'
```
Comment (FabianWilms) [inline docs/components/muc-banner.md:2]: I normally don't like mixing different tasks in a single PR but I will let this change pass.

**Review by FabianWilms — CHANGES_REQUESTED** [review]

File: src/components/index.ts (hunk 1 above)
Comment (langehm) [reply src/components/index.ts:4]: Should we change the import "format" from the default-export of those specific vue-files/components to the module-import of the `index.ts` file inside those folders?

Comment (FabianWilms) [reply src/components/index.ts:4]: Yes I think that would look cleaner!

Comment (langehm) [reply src/components/index.ts:4]: Issue above is resolved and import is changed to use moduled import for all known components until now.

**Review by FabianWilms — APPROVED** [review]

File: docs/components/muc-banner.md (hunk 2 above)
Comment (langehm) [reply docs/components/muc-banner.md:2]: This was necessary to satisfy the linter.
`Component name "Basic" should always be multi-word  vue/multi-word-component-names`

**Pull request merged**